docker-compose.yaml

```diff
@@ -17,7 +17,9 @@ services:
         build:
             context: ./docker/caddy/
         working_dir: /srv/
-        volumes: ['./backend/:/srv/']
+        volumes:
+            - './backend/:/srv/'
+            - './frontend/build/caddy/:/data/caddy/'
```

Comment:
Goal here is to share the /data/caddy dir to the local frontend dir, and then reuse it from the frontend container